function.tf

```diff
@@ -0,0 +1,78 @@
+locals {
+  enable_function = var.enable_function || var.enable_extensions
+}
+
+resource "google_service_account" "cloud_functions" {
```

Comment:
Why functions (plural) instead of function? This is used by one function.

Reply:
i am not sure, this was a copy-paste. it should be fine to re-create the resource so i'll rename it